akd_mysql/src/mysql.rs

```diff
@@ -911,6 +911,10 @@ impl Storage for AsyncMySqlDatabase {
         )
         .await;
 
+        if St::data_type() == StorageType::TreeNode {
+            println!("Get direct tree node.");
+        }
+
```

Comment:
Did you intend to remove this?

Reply:
Good catch! The byproduct of fixing the cache issue was that this message didn't appear anymore, so was missed. Thanks for pointing out!